### PR TITLE
New gom player cheat sheet

### PR DESCRIPTION
**Pull request by jebin95**
created gom player cheat sheet

--------
ia: https://duck.co/ia/view/gom_player_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@jebin95 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gautamkrishnar** [comment]
@jebin95 welcome to duckduckhack :small_blue_diamond: :smile:

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+  "id": "gom_cheat_sheet",
+  "name": "gomplayer",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gom-player.json]: change this to `"name": "GOM Player",`

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  "sections": {
+    "Exit": [
+      {
+        "key": "[Alt] [F4]",
+        "val": "Exit Gomplayer"
+      }
+    ],
+    "Open": [
+      {
+        "key": "[F2]",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gom-player.json]: remove [] for single keys
change this to `"key": "F2",`

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        "val": "Open File"
+      },
+      {
+        "key": "[F3]",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gom-player.json]: change this to `"key": "F3",`

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        "val": "Open Subtitles"
+      },
+      {
+        "key": "[F4]",
+        "val": "Open the Prefences window"
+      }
+    ],
+    "Playlist": [
+      {
+        "key": "[F8]",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gom-player.json]: also this

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    "Open DVD": [
+      {
+        "key": "[F12]",
+        "val": "open DVD"
+      }
+    ],
+    "Skins": [
+      {
+        "key": "[K]",
+        "val": "Choose skin"
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gom-player.json]: capitalize S

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+    "Playlist": [
+      {
+        "key": "[F8]",
+        "val": "Open/Close Playlist window"
+      }
+    ],
+    "Open DVD": [
+      {
+        "key": "[F12]",
+        "val": "open DVD"
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gom-player.json]: capitalize O

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+      }
+    ],
+    "Skins": [
+      {
+        "key": "[K]",
+        "val": "Choose skin"
+      },
+      {
+        "key": "[Alt] [F10]",
+        "val": "Open Skin Manager"
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gom-player.json]: keep following the same capitalization scheme always:
eg you can use:
"val": "Open skin manager"
or
"val": "Open Skin Manager"
here...
You need to update all vals to match the same scheme. I would recommend you to capitalize only the first char of the key.

**Comment by gautamkrishnar** [comment]
@jebin95 you could add some more shortcuts too

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 8]
```diff
@@ -0,0 +1,68 @@
+{
+  "id": "gom_cheat_sheet",
+  "name": "gomplayer",
+  "description": "Multimedia Player",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/gom-player.json]: @jebin95 "Keyboard shortcuts for GOM Player, a media player for Windows developed by the Gretech Corporation"

lets cange the desc to above

**Comment by moollaza** [comment]
@jebin95 I've corrected the ID of your IA Page to `gom_player_cheat_sheet`. Please update the `id` in your JSON to match: ` gom_player_cheat_sheet `

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=gom player cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/gom_player_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 9]
```diff
@@ -0,0 +1,68 @@
+{
+  "id": "gom_player_cheat_sheet",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/gom-player.json]: Could you correct the indentation to be 4 spaces per level throughout.

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+        "key": "F2",
+        "val": "Open File"
+      },
+      {
+        "key": "F3",
+        "val": "Open Subtitles"
+      },
+      {
+        "key": "F4",
+        "val": "Open the Prefences window"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/gom-player.json]: That a typo, or is it supposed to be `Prefences`?

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 11]
```diff
@@ -0,0 +1,68 @@
+{
+  "id": "gom_player_cheat_sheet",
+  "name": "GOM Player",
+  "description": "Keyboard shortcuts for GOM Player, a media player for Windows developed by the Gretech Corporation",
+  "metadata": {
+    "sourceName": "Gom Lab",
+    "sourceURL": "http://player.gomlab.com/guide.gom?tab=51"
+  },
+  "aliases": [
+    "gom"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/gom-player.json]: We could have `gom media player` too.

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+    "Playlist": [
+      {
+        "key": "F8",
+        "val": "Open/Close Playlist window"
+      }
+    ],
+    "Open DVD": [
+      {
+        "key": "F12",
+        "val": "Open DVD"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/gom-player.json]: How about moving this into the `Open` section?

File: share/goodie/cheat_sheets/json/gom-player.json [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+      },
+      {
+        "key": "F4",
+        "val": "Open the Prefences window"
+      }
+    ],
+    "Playlist": [
+      {
+        "key": "F8",
+        "val": "Open/Close Playlist window"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/gom-player.json]: We could probably move this into the `Open` section too.

**Comment by edgesince84** [comment]
@jebin95,  thanks again for creating the Gom Player Cheat Sheet! You’ve been doing a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in Gom Player.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to media players

OR

- Test live <topic> Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=gom player cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/gom_player_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=gom player cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/gom_player_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@jebin95 Thanks! This LGTM :+1:

It looks like @GuiltyDolphin has left you some suggested improvements that would be good to consider. It would be great for you to share this with the GOM Player users/community to see what they think!

**Comment by daxtheduck** [comment]
@jebin95 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**